inspirehep/modules/search/__init__.py

```diff
@@ -27,5 +27,8 @@
 from __future__ import absolute_import, print_function
 
 from .ext import INSPIRESearch
+from .query_factory import inspire_query_factory
 
-__all__ = ('INSPIRESearch', )
+IQ = inspire_query_factory()
```

Comment:
You are right. I was actually more worried about the loading timing, but I see now that this is just a closure anyway, so we should be good on that case too.